### PR TITLE
Extract duplicate processing to handleColorChange method

### PR DESCRIPTION
**Pull request by akameco**

### CODE REVIEW

File: app/src/main/java/net/nashihara/naroureader/fragments/SettingsFragment.java [hunk 1]
```diff
@@ -31,29 +31,24 @@ public void onCreatePreferences(Bundle bundle, String s) {
...
-                getFragmentManager(),
-                background,
-                color -> pref.edit().putInt(getString(R.string.body_background), color).apply());
+            handleColorChange(R.string.body_background, R.color.colorBackground);
         }
 
         return super.onPreferenceTreeClick(preference);
     }
+
+    private void handleColorChange(int target, int defaultColor) {
```
Comment (nshiba) [inline app/src/main/java/net/nashihara/naroureader/fragments/SettingsFragment.java]: this can use annotaions. 
e.g.) `@StringRes` , `@ColorRes`

Comment (nshiba) [inline app/src/main/java/net/nashihara/naroureader/fragments/SettingsFragment.java]: I think that the name "stringResId" and "colorResId" are better.

**Review by nshiba — CHANGES_REQUESTED** [review]

**Comment by akameco** [comment]
@nshiba Updated PR with your suggestions. Thanks for the assist 👍

**Review by nshiba — APPROVED** [review]

**Comment by nshiba** [comment]
greate! thanks!

**Pull request merged**